stdlib/influxdata/influxdb/alerts/alerts.flux

```diff
@@ -18,6 +19,14 @@ from = (start, stop=now(), fn) =>
 // Log records notification events
 log = (tables=<-) => tables |> to(bucket: bucket)
 
+// Notify will call the endpoint and log the results.
+notify = (tables=<-, endpoint, data={}) =>
+    tables
+        |> experimental.set(o: data)
+        |> experimental.group(mode: "extend", columns: experimental.objectKeys(o: data))
+        |> endpoint()
+        |> log()
```

Comment:
We need to add some columns here:

* `_sent` - this one of `true` or `false` as a string (it needs to be a tag). Maybe the endpoint needs to do this part? 
* `_status_timestamp` - this should be the `_time` of the original data
* `_time` - this should be updated to be `now()`. I.e. record the time the notification ran.
* `_error` - this should be the error message if any. must be set when `_sent` is false. Same question here maybe the endpoint needs to do this part.

Reply:
There's a similar value for all of those that show up in the original spec and they are all properties of the endpoint. While the spec needs to be updated to the rfc, I think it makes sense that those properties would all be part of the endpoint.

Reply:
The timestamps should be handled as part of `notify` though as that is not something the endpoint needs to be aware of. So that means `_sent` and `_error` should be handled by the endpoint and `_time` and `_status_timestamp` should be handled in `notifty`. Does that sound good?

Reply:
`_time` should definitely be handled by the endpoint as it is the time that the endpoint sent the message. It's not really related to `notify()` IMO because notify isn't actually sending anything.

We could potentially duplicate the `_status_timestamp` column though in notify so that `endpoint()` only has to worry about adding `_time`. I think that would make sense.